src/Components/Seen/SeenWindow.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import GeneralWindow from 'Components/GeneralComponents/GeneralWindow';
+
+export default function SeenWindow({ unmountSeenWindow }) {
+	return (
+		<GeneralWindow unmountGeneralWindow={unmountSeenWindow} rightSided={true}>
```

Comment:
please change rightSided={true} to rightSided

Reply:
Done.